pkg/wamr/Makefile.dep

```diff
@@ -0,0 +1,5 @@
+USEMODULE += sema
+
+#WAMR supports "X86_32/64", "AARCH64", "ARM", "THUMB", "MIPS" and "XTENSA"
+#no 8/16 Bit TODO: might need to blacklist more
+FEATURES_BLACKLIST += arch_8bit arch_16bit
```

Comment:
```suggestion
FEATURES_REQUIRED_ANY += arch_native|arch_arm|arch_mips32r2|arch_esp
```

IMO it is easier to maintain a list of supported architectures than a list of unsupported. Also, this would give a more helpful error message (as it will tell the user which of her board she could try instead, rather than just "meh, I don't like this one").

No support for RISC-V yet, even when not using JIT?

Reply:
atm the pkg Makefile translates like this 
```
ifeq ($(CPU),native)
#$(CPU) is defined for every CPU
#everyone build on x86_32
  WAMR_BUILD_TARGET = X86_32
else ifeq ($(findstring arm,$(CPU_ARCH)),arm)
  WAMR_BUILD_TARGET = THUMB
else ifeq ($(CPU_ARCH),mips32r2)
  WAMR_BUILD_TARGET = MIPS
else ifeq ($(CPU_ARCH),xtensa)
  WAMR_BUILD_TARGET = XTENSA
endif
```

Reply:
i will check if upstream hast risc-v

Reply:
thanks to downloading master that check was easy :)

Reply:
will be added